runtime/lib/app-runtime.js

```diff
@@ -489,7 +511,7 @@ AppRuntime.prototype.resetNetwork = function resetNetwork (options) {
   return Promise.all([
     this.component.lifetime.destroyAll(),
     this.setMicMute(false, { silent: true })
-  ]).then(() => this.component.custodian.resetNetwork(options))
+  ]).then(() => this.component.custodian.reConfigureNetwork())
```

Comment:
Method name suggestion:

```suggestion
  ]).then(() => this.component.custodian.reconfigureNetwork())
```